packages/common/src/types/GSNContractsDataTypes.ts

```diff
@@ -20,7 +20,7 @@ export const Deposited: EventName = 'Deposited'
 /**
  * Emitting any of these events is handled by GSN clients as a sign of activity by a RelayServer.
  */
-export const ActiveManagerEvents = [RelayServerRegistered, RelayWorkersAdded, TransactionRelayed, TransactionRejectedByPaymaster]
+export const ActiveManagerEvents = [RelayWorkersAdded, TransactionRelayed, TransactionRejectedByPaymaster]
```

Comment:
I think with a new relay discovery, this event grouping is redundant, isn't it?

Reply:
no, but this event is no longer a RelayHub event, but RelayRegistrar.